Source/ACE/Entity/Container.cs

```diff
@@ -51,14 +52,54 @@ public Container(AceObject aceObject, ObjectGuid guid)
         {
         }
 
+        public virtual void ContainerPlacement(WorldObject inventoryItem)
+        {
+            var removedItem = inventory.Remove(inventoryItem.Guid);
+            var invList = inventory.OrderBy(o => o.Value.Placement);
+            uint place = 0;
+            AceObject saveableCopy;
+            foreach (var keyValuePair in invList)
```

Comment:
I figured that's what it was, but it's coded rather oddly.  I don't think we need to be saving on every ordering alteration.  We can let the periodic save handle this sort of stuff.